docs/dom-testing-library/api-configuration.mdx

```diff
@@ -130,6 +130,34 @@ option.
 screen.getByTestId('foo', {suggest: false}) // will not throw a suggestion
 ```
 
+> **Note**
```

Comment:
You can use a [Admonition](https://docusaurus.io/docs/markdown-features/admonitions) for this section, it should look better with it.

```suggestion
> :::note
```